modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * registry/installing-openstack- .adoc
+// * registry/configuring-registry-operator.adoc
+// * registry/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-configuring-registry-storage-swift-trust_{context}"]
+= Configuring the Image Registry Operator to trust Swift storage
+
+You must configure the Image Registry Operator to trust OpenStack Swift storage.
+
+////
+...to allow the client to pull the image layers from the image registry rather than from links directly from Swift.
+////
+
+.Procedure
+
+* To configure the Image Registry Operator to trust OpenStack Swift storage, change the ____ field on the `config.imageregistry` object to `spec.disableRedirect: true`.
+
++
+[source,terminal]
```

Comment:
According to https://bugzilla.redhat.com/show_bug.cgi?id=1810461#c13, the following should be all that is needed:

```
$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"disableRedirect":"true"}}'
```

Reply:
@mandre As in that command should be almost the entirety of this PR? Easy enough, if so. 

Any thoughts on the second part of this comment? https://bugzilla.redhat.com/show_bug.cgi?id=1810461#c14

Reply:
> @mandre As in that command should be almost the entirety of this PR? Easy enough, if so.
> 
> Any thoughts on the second part of this comment? https://bugzilla.redhat.com/show_bug.cgi?id=1810461#c14

If I understand comment 14 correctly what we're doing here (disable redirect) is what the customer has confirmed fixes the issue. I do not think we want to document the other suggested solution as it's a lot more complex and forces the nodes to trust the CA ultimately (while we trust it only when initiating connections to OpenStack otherwise).